app/models/spot.rb

```diff
@@ -48,7 +48,24 @@ def create_matches
                            .where.not(user_id: user_id)
 
     compatible_spots.each do |spot|
-      matches.create!(first_user_id: user_id, second_user_id: spot.user_id)
+      create_match(spot)
     end
   end
+
+  def create_match(spot)
+    match = matches.create!(first_user_id: user_id, second_user_id: spot.user_id)
+    notify(match, match.second_user)
```

Comment:
It's not necessary to pass the `match.second_user` because you're passing the match